app/uk/gov/hmrc/tariffclassificationfrontend/views/assigned_cases.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, Queue}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials._
+@import uk.gov.hmrc.tariffclassificationfrontend.views.AssignedCases
+@import uk.gov.hmrc.tariffclassificationfrontend.views.AssigneeCount
+
+@(queues: Seq[Queue], cases: Seq[Case], assigneeId: Option[String])(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
```

Comment:
How large might the collection os cases be in production? Will this view be slow if there are 1000's of cases returned form the backend?

Reply:
We are only getting the 'live' cases which are currently assigned to someone.  E.g. 30 ops with 20 live cases each is only 600 cases.

Reply:
@retroredge I think if performance becomes an issue we can do a fix later on, given the queues currently have no paging at all and seem okay (ish) we should be okay

Reply:
I agree we should have pagination